src/api/buildPayTransaction.ts

```diff
@@ -1,37 +1,50 @@
-import { base } from 'viem/chains';
-import { PAY_HYDRATE_CHARGE } from '../network/definitions/pay';
-import { sendRequest } from '../network/request';
-import { PAY_UNSUPPORTED_CHAIN_ERROR_MESSAGE } from '../pay/constants';
+import {
+  CDP_CREATE_PRODUCT_CHARGE,
+  CDP_HYDRATE_CHARGE,
+} from '../network/definitions/pay';
+import { type JSONRPCResult, sendRequest } from '../network/request';
```

Comment:
nit: we've been separating importing types. Unless this is what the formatter now prefers

Reply:
i just ran `yarn check`